src/Step/Deploy/PreDeploy/ConfigUpdate/Cache.php

```diff
@@ -67,8 +76,19 @@ public function execute()
 
         if (isset($cacheConfig['frontend'])) {
             $cacheConfig['frontend'] = array_filter($cacheConfig['frontend'], function ($cacheFrontend) {
-                return $cacheFrontend['backend'] !== 'Cm_Cache_Backend_Redis'
-                    || $this->testRedisConnection($cacheFrontend['backend_options']);
+                $backend = stripslashes($cacheFrontend['backend']);
+                $this->checkBackendModel($backend);
+
+                if (!in_array($backend, CacheFactory::AVAILABLE_REDIS_BACKEND, true)) {
+                    return true;
+                }
+
+                $backendOptions = ($backend === CacheFactory::REDIS_BACKEND_REMOTE_SYNHRONIZED_CACHE)
+                        ? $cacheFrontend['backend_options']['remote_backend_options']
+                        : $cacheFrontend['backend_options'];
+
+                return !in_array($backend, CacheFactory::AVAILABLE_REDIS_BACKEND, true)
```

Comment:
Looks like you've already done this check `!in_array($backend, CacheFactory::AVAILABLE_REDIS_BACKEND, true)` few lines above, why do you need to do it again?

Reply:
Thanks, I forgot to remove this.